src/lua/init.lua

```diff
@@ -203,7 +203,11 @@ rawset(package, "search", search)
 rawset(package, "searchroot", searchroot)
 rawset(package, "setsearchroot", setsearchroot)
 
+local compat = require('compat')
+package.loaded.compat = nil
```

Comment:
I don't see any benefit from having `internal.compat` globally accessible. If we are fine with littering global namespace, maybe just leave compat there. AFAIK, there is nothing wrong with having `require 'compat'` instead of `require('tarantool').compat`.

Reply:
To be precise.

Now:

```lua
local compat = _require('compat') <- src/lua/compat.lua
_require('internal.compat') <- src/lua/compat.c

require('tarantool').compat = compat
package.loaded.compat = nil
```

Proposal:

```lua
local compat = _require('internal.compat') <- src/lua/compat.lua
_require('internal.compat_luac') <- src/lua/compat.c

require('tarantool').compat = compat
```

Quorum decision (with @igormunkin): let's leave it as is, because renaming of compat.c to compat_luac seems losely motivated.